### PR TITLE
Add tests for certain visuals

### PR DESCRIPTION
**Pull request by astrofrog**
I'm hoping to reproduce some of the issues in e.g. #1276 and #1277.

Should be rebased once #1278 is merged

### CODE REVIEW

File: vispy/visuals/tests/test_line.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+"""
+Tests for AxisVisual
```
Comment (rougier) [inline vispy/visuals/tests/test_line.py]: Title needs to be changed.

File: vispy/visuals/tests/test_line.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+from vispy.scene import visuals
+from vispy.testing import (requires_application, TestingCanvas,
+                           run_tests_if_main)
+
+
+@requires_application()
+def test_line():
+    with TestingCanvas() as c:
+        axis = visuals.Line(pos=[[-1.0, 0], [1.0, 0]])
```
Comment (rougier) [inline vispy/visuals/tests/test_line.py]: `line = ` maybe ?

**Comment by astrofrog** [comment]
For some reason I couldn't get AppVeyor to fail in the same way as the failures I've seen, so I'm going to try and add some more tests. This is not ready for merging!

File: vispy/visuals/tests/test_line.py (hunk 1 above)
Comment (astrofrog) [inline vispy/visuals/tests/test_line.py]: Fixed

File: vispy/visuals/tests/test_line.py (hunk 2 above)
Comment (astrofrog) [inline vispy/visuals/tests/test_line.py]: Fixed

**Comment by Eric89GXL** [comment]
@astrofrog where are we here?

**Comment by astrofrog** [comment]
Sorry I dropped the ball on this, I've copied some of the tests here into https://github.com/vispy/vispy/pull/1743 so I'll just go ahead and close this.

**Pull request closed without merging**